frontend/public/components/utils/documentation.tsx

```diff
@@ -1,15 +1,98 @@
-// Prefer the documentation base URL passed as a flag, but fall back to the latest docs if none was specified.
-export const openshiftHelpBase =
-  window.SERVER_FLAGS.documentationBaseURL || 'https://docs.okd.io/latest/';
+const UPSTREAM_LATEST = 'https://docs.okd.io/latest/';
+
+// Prefer the documentation base URL passed as a flag, but fall back to the latest upstream docs if none was specified.
+export const openshiftHelpBase = window.SERVER_FLAGS.documentationBaseURL || UPSTREAM_LATEST;
+
+export const DOC_URL_OPERATORFRAMEWORK_SDK = 'https://sdk.operatorframework.io/';
+export const DOC_URL_PODDISRUPTIONBUDGET_POLICY = `${UPSTREAM_LATEST}rest_api/policy_apis/poddisruptionbudget-policy-v1.html#poddisruptionbudget-policy-v1`;
+export const DOC_URL_PODMAN = 'https://podman.io/';
+export const DOC_URL_RED_HAT_MARKETPLACE =
+  'https://marketplace.redhat.com/en-us?utm_source=openshift_console';
+
+const KUBE_DOCS = 'https://kubernetes.io/docs/';
+export const DOC_URL_STORAGE_CLASSES_AWS_EBS = `${KUBE_DOCS}/concepts/storage/storage-classes/#aws-ebs`;
+export const DOC_URL_STORAGE_CLASSES_AZURE_DISK = `${KUBE_DOCS}/concepts/storage/storage-classes/#azure-disk`;
+export const DOC_URL_STORAGE_CLASSES_AZURE_FILE = `${KUBE_DOCS}/concepts/storage/storage-classes/#azure-file`;
+export const DOC_URL_STORAGE_CLASSES_GCE = `${KUBE_DOCS}/concepts/storage/storage-classes/#gce`;
+export const DOC_URL_STORAGE_CLASSES_GLUSTERFS = `${KUBE_DOCS}/concepts/storage/storage-classes/#glusterfs`;
+export const DOC_URL_STORAGE_CLASSES_LOCAL = `${KUBE_DOCS}/concepts/storage/storage-classes/#local`;
+export const DOC_URL_STORAGE_CLASSES_OPENSTACK_CINDER = `${KUBE_DOCS}/concepts/storage/storage-classes/#openstack-cinder`;
+export const DOC_URL_STORAGE_CLASSES_PORTWORX_VOLUME = `${KUBE_DOCS}/concepts/storage/storage-classes/#portworx-volume`;
+export const DOC_URL_STORAGE_CLASSES_QUOBYTE = `${KUBE_DOCS}/concepts/storage/storage-classes/#quobyte`;
+export const DOC_URL_STORAGE_CLASSES_SCALEIO = `${KUBE_DOCS}/concepts/storage/storage-classes/#scaleio`;
+export const DOC_URL_STORAGE_CLASSES_STORAGEOS = `${KUBE_DOCS}/concepts/storage/storage-classes/#storageos`;
+export const DOC_URL_STORAGE_CLASSES_VSPHERE = `${KUBE_DOCS}/concepts/storage/storage-classes/#vsphere`;
+
+export const documentationURLs = {
```

Comment:
nit: maybe type the variable as well?
```suggestion
type documentationURLsTypes = {
  [key: string]: docURLs
}

export const documentationURLs: documentationURLsTypes = {
```